python/pyspark/ml/tests.py

```diff
@@ -684,12 +684,32 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def _compare_params(self, m1, m2, param):
+        """
+        Compare 2 ML params, assert they have the same param. The param must be a parameter of m1.
+        """
+        # Prevent key not found error in case of some param neither in paramMap and
```

Comment:
"neither in paramMap and defaultParamMap" --> "in neither paramMap nor defaultParamMap"